oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/storage/InMemoryModule.java

```diff
@@ -36,7 +39,24 @@ protected void configure() {
     bind(SessionSecurity.class).toInstance(resourceOwnerRepository);
 
     InMemoryClientRepository clientRepository = new InMemoryClientRepository();
-    clientRepository.register(new ClientRegistrationRequest("857613db7b18232c72a5093ad19dbc6df74a139e", "test", Collections.singleton("http://localhost:8080/oauth/callback")));
+    
+    String registeredRedirectURI = "https://httpbin.org/get";
```

Comment:
Redirect URL looks strange.

Reply:
It is just for the example, it redirects to a page that displays all parameters on a page, so anyone can try example oauth server alone (without creating another server to be redirected to), without modifications.

Reply:
Changes removed, anyway